images/dind/node/Dockerfile

```diff
@@ -25,7 +25,12 @@ RUN dnf -y update && dnf -y install\
  python-netaddr\
  python2-pyroute2\
  python2-requests\
- PyYAML
+ PyYAML\
+ cri-o
+
+# Remove the CRI-O CNI network configs so openshift-sdn's will be used instead
+RUN rm -f /etc/cni/net.d/100-crio-bridge.conf
+RUN rm -f /etc/cni/net.d/200-loopback.conf
```

Comment:
Stylistically I think it's preferred to reduce the number of RUN commands when it's easy to do so, for efficiency. (So, eg, call `rm` once with two paths)